createAccountCore/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
You don't need this file or the `consumer-rules` file

Reply:
Deleted.